screen.html

```diff
@@ -107,4 +108,81 @@
             autocomplete="off" spellcheck="false" tabindex="-1"></textarea>
     </div>
     <div id="noVNC_identify_monitor">0</div>
+
+    <div id="mySidenav" class="loadin">
```

Comment:
The side nav on the second screen appears to be in dark mode, while the primary screen is in light mode.

Reply:
The secondary screen has no knowledge of what the primary screen mode has in KasmWeb. I went with the dark mode version because it was closer to the primary KasmVNC. It might be possible to pass that down I guess, I'll look into it.

Reply:
This should now be fixed. I'm checking the localstorage `theme` setting that is set in kasmweb and updating colours accordingly, if that's not set it defaults to the dark version.